storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -139,6 +139,9 @@
     @isString
     public static final String STORM_META_SERIALIZATION_DELEGATE = "storm.meta.serialization.delegate";
 
+    @isType(type=List.class)
```

Comment:
It is system-wide, or per node depending how you look at it (i.e. storm.conf could be different across machines).